src/platforms/hubspot/schemas.ts

```diff
@@ -96,6 +96,12 @@ export const baseHubspotObjectSchema = <T extends z.ZodSchema>(properties: T) =>
     })
     .passthrough();
 
+export const upsertResponseSchema = z
```

Comment:
we aren't using passthrough anymore for responses

Reply:
gotcha — why did we decide on that btw?